Shopify/hydrogen
# [DO NOT MERGE] Example hydrogen on cloudflare edge

This is an example of an Hydrogen setup for cloudflare edge.

### WHY are these changes introduced?
Reports of people having issues deploying to cloudflare. Our guide is too simple, it actually takes a while to set this up

### WHAT is this pull request doing?
This PR is mainly to get feedback. 

* Two options for sharing:
  * [recommended] I can keep it in my own repo: https://github.com/lordofthecactus/hydrogen-v2-cloudflare-edge, which will just freeze it as an example. 
  * We can add an `examples` folder but this adds to our repo

### HOW to test your changes?
1. Follow the readme inside `examples/cloudflare-edge`
2. Check that `install`, `dev` works
3. If possible, create a cloudflare account and make a deploy with `npm run deploy`
